src/ACEtk/details/verify/Positive/src/assignment.hpp

```diff
@@ -0,0 +1,18 @@
+template< typename Arg = std::initializer_list< typename Range::value_type >,
+          utility::Require< false, IsPositive, Arg > = true,
+          utility::Require< true, std::is_assignable, Range, Arg > = true,
+          utility::Require< true, std::is_constructible, Range, Arg > = true
+        >
+Positive& operator=( Arg arg ){
+  static_cast< Range& >( *this ) = check( std::forward< Arg >( arg ) );
+  return *this;
+}
+
+template< typename Arg,
+          utility::Require< true, IsPositive, Arg > = true,
+          utility::Require< false, std::is_assignable, Range&, Arg > = true
```

Comment:
Here you require that the argument is not assignable to the range and then attempt to assign to it